flink-table/flink-table-planner/src/main/java/org/apache/flink/table/planner/plan/nodes/exec/stream/StreamExecLocalGroupAggregate.java

```diff
@@ -147,7 +147,7 @@ protected Transformation<RowData> translateToPlanInternal(
                         true);
         generator.needAccumulate().needMerge(0, true, null);
         if (needRetraction) {
-            generator.needRetract();
+            generator.needRetract(aggCallNeedRetractions);
         }
```

Comment:
1. why global agg doesn't need retract method?  because agg when enable two stage. the global agg doesn't receive retract msg, so only local agg using retract
2. why window WTF doesn't need retract method? because Window WTF can not process update msg now. it will supports later
3. why group window agg doesn't need retract method? because the group window agg doesn't support two stage in flink, but supports on blink